routes/historical_persons.py

```diff
@@ -128,13 +140,13 @@ def put_new_hisrorical_person():
 @returns_json
 def post_hisrorical_person_by_id(object_id):
     if HistoricalPerson.query.get(object_id) is None:
-        abort(404, 'Historical person not found')
+        abort(400, "Historical person with id = %s not found" % object_id)
```

Comment:
If we pass some id in url, I think it will be better to abort with 404 code (page not found), in other case - when some entity with wrong id passed in body - abort with 400 code.